doc/adr/0010-js-purge.md

```diff
@@ -0,0 +1,67 @@
+# 10. js-purge
+
+Date: 2021-06-30
+
+## Status
+
+Accepted
+
+## Context
+
+JetStream provides the ability to purge streams by sending a request message to:
+`$JS.API.STREAM.PURGE.<streamName>`. The request will return a new message with
+the following JSON:
+
+```typescript
```

Comment:
be better to show json here

Reply:
The syntax showing things like: `{foo?: string, bar: number}` is fairly intuitive, and conveys that field `foo` is optional, while `bar` is not - the `typescript` directive colors it properly.